src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -114,12 +114,10 @@ public function __invoke(RequestEvent $event)
     /**
      * Attempts to switch to another user.
      *
-     * @return TokenInterface|null The new TokenInterface if successfully switched, null otherwise
```

Comment:
the description seems value to be kept. maybe as a normal description with `@return`